pypi_count/py_counter.py

```diff
@@ -83,3 +90,18 @@ def count_classes_with_docstring(self):  # pylint: disable=R1710
                 total.append(count)
                 count += 1
         return count
+
+    def count_function_parameters(self):
```

Comment:
This function should take in the name of the function to count the parameters of.

Reply:
@caldwella2